framework/include/media/MediaRecorder.h

```diff
@@ -191,7 +191,7 @@ class MediaRecorder
 
 	/**
 	 * @brief Set limitation of recording time by given value(second), will be stopped when it reaches that.
-	 * This should be called after create but before prepare
+	 * This should be called after setDataSource
```

Comment:
setDuration is calculated in prepare phase, 2 things we can do that.
1. simply, change description to "after setDataSource but before prepare"
2. Or, separate code regarding setDuration from prepareRecorder and make it's own function in impl class
    But description should be modified to "Before start record, set limitation of recording data...blahblah... This should be called after setDataSource(As you modified)"